CharlieBackend.AdminPanel/Controllers/CoursesController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using CharlieBackend.AdminPanel.Services.Interfaces;
+using CharlieBackend.AdminPanel.Utils.Interfaces;
+using CharlieBackend.Core.DTO.Course;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.AdminPanel.Controllers
+{
+    [Route("[controller]/[action]")]
+    public class CoursesController : Controller
+    {
+        private readonly ICourseService _courseService;
+
+        public CoursesController(ICourseService courseService)
+        {
+            _courseService = courseService;
+        }
+
```

Comment:
did you foget [HttpGet] attribute for this method?

Reply:
Yes, thnks. Solved here https://github.com/ita-social-projects/WhatBackend/pull/377/commits/e734b46153858d0c1699c72e35e6bbc34002b804.